sql/0_0_1-to-1_0_0_upgrade.sql

```diff
@@ -71,6 +71,38 @@
 --  #IfNotListOccupation
 --    Custom function for creating Occupation List
 
+#IfTable
+RENAME `openemr_module_vars` TO `libreehr_module_vars`;
```

Comment:
Doesn;t the syntax require the word TABLE?   As in
RENAME TABLE `openemr_module_vars` TO `libreehr_module_vars`;